server/src/test/java/org/elasticsearch/action/admin/cluster/snapshots/SnapshotBlocksIT.java

```diff
@@ -176,4 +177,30 @@ public void testSnapshotStatusWithBlocks() {
             setClusterReadOnly(false);
         }
     }
+
+    public void testSnapshotWithDateMath() {
```

Comment:
I'm not sure what this is exactly testing. There are no assertions that check that the date math functionality has worked in a meaningful way. Also why is this testing blocks?

Reply:
it tests that it is possible to create snapshot with name `<snapshot-{now/d}>` and it is resolved into smth like `snapshot-2018.05.11` using `nameExpressionResolver.resolveDateMathExpression(snapshotName)` - I assume that `nameExpressionResolver` works.

Agree - it is worth to move test to `DedicatedClusterSnapshotRestoreIT` - is that ok ?